zerver/models.py

```diff
@@ -480,6 +482,9 @@ def name_changes_disabled(realm: Optional[Realm]) -> bool:
         return settings.NAME_CHANGES_DISABLED
     return settings.NAME_CHANGES_DISABLED or realm.name_changes_disabled
 
+def avatar_changes_disabled(realm: Realm) -> bool:
+    return settings.AVATAR_CHANGES_DISABLED or realm.avatar_changes_disabled
```

Comment:
No check for `realm is None` has to be done here, as avatar is not uploaded when registration is called during realm creation.